client/my-sites/checkout/cart/style.scss

```diff
@@ -166,7 +162,7 @@
 		form {
 			display: block;
 			border-top: 1px solid var( --color-border-subtle );
-			margin: 15px -15px 0 -15px;
+			margin: 15px -15px 0;
```

Comment:
This was also in another PR of yours. Might be a conflict down the line

Reply:
Yeah, I'll rebase before I commit this one to make sure. :)